openmetadata-ui/src/main/resources/ui/src/utils/CommonUtils.tsx

```diff
@@ -498,12 +498,12 @@ export const getEntityPlaceHolder = (value: string, isDeleted?: boolean) => {
 export const getEntityDeleteMessage = (entity: string, dependents: string) => {
   if (dependents) {
     return t('message.permanently-delete-metadata-and-dependents', {
-      entityName: getTitleCase(entity),
+      entityName: entity,
```

Comment:
Why we removed `getTitleCase` here?

Reply:
getTitleCase doesnt handle well the case where the entity name has a dot(.) in it, it removes the dot and title cases the entity name. With this change, it will allow the user to see the entityName as it is